tests/chains/ethereum/contracts/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "contracts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "TBD",
```

Comment:
"main" and "scripts" are unnecessary.
"license" should be "Apache-2.0".

Reply:
Add license
https://github.com/datachainlab/yui-relayer-build/pull/24/commits/b7011d80c90f3c54f4f7f76106637736b312d228